core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue

```diff
@@ -144,6 +156,27 @@ export default Vue.extend({
         .filter((param) => param.value !== 0)
         .map((parameter) => decode(parameter.name, parameter.value))
     },
+    // DEV_ID params do not exist yet for temperature sensors, so here we detect the incoming message instead
+    celsius_temperature(): number | undefined {
+      return mavlink_store_get(mavlink, 'SCALED_PRESSURE3.messageData.message.temperature') as number / 100.0
+    },
+    celsius(): deviceId[] {
+      if (!this.celsius_temperature) {
+        return []
+      }
+      return [
+        {
+          bus: 1,
+          paramValue: 0,
+          deviceIdNumber: 0,
+          devtype: 0,
+          busType: 'I2C',
+          address: '77',
+          deviceName: 'Celsius',
```

Comment:
```suggestion
          deviceName: 'TSYS01 (Celsius)',
```
I’m unsure about this, but since the other sensors specify the actual sensor type then it might make more sense?

Reply:
yeah, I'll do this and then a follow-up pr to add user-friendly names on Sub I think